arsceneview/build.gradle.kts

```diff
@@ -100,17 +67,63 @@ mavenPublishing {
     signAllPublications()
 }
 
-if (project.properties["filamentPluginEnabled"]?.toString()?.toBoolean() == true) {
-    configure<io.github.sceneview.FilamentToolsPluginExtension> {
-        // Material generation: .mat -> .filamat
-        materialInputDir.set(project.layout.projectDirectory.dir("src/main/materials"))
-        materialOutputDir.set(project.layout.projectDirectory.dir("src/main/assets/materials"))
+dokka {
+    moduleName.set("SceneView Android")
+    dokkaSourceSets.configureEach {
+        externalDocumentationLinks.register("example-docs") {
+            url("https://example.com/docs/")
+            packageListUrl("https://example.com/docs/package-list")
+        }
     }
+    dokkaSourceSets.main {
+        includes.from("README.md")
+        sourceLink {
+            localDirectory.set(file("src/main/kotlin"))
+            remoteUrl("https://example.com/src")
+            remoteLineSuffix.set("#L")
+        }
+        reportUndocumented.set(true)
+        skipEmptyPackages.set(true)
+        skipDeprecated.set(true)
+        jdkVersion.set(21)
+
+        // Do not create a link to the online Android SDK documentation
+        enableAndroidDocumentationLink.set(false)
 
-    tasks.named("clean") {
-        doFirst {
-            delete("src/main/assets/materials")
-            delete("src/main/assets/environments")
+        // AndroidX + Compose docs
+        externalDocumentationLinks {
+            register("developer.android.com/reference") {
+                url("https://developer.android.com/reference/")
+                packageListUrl("https://developer.android.com/reference/androidx/package-list")
+            }
+            register("developer.android.com/reference/kotlin") {
+                url("https://developer.android.com/reference/kotlin/")
+                packageListUrl("https://developer.android.com/reference/kotlin/androidx/package-list")
+            }
         }
+
+        sourceLink {
+            localDirectory.set(project.file("src/main/kotlin"))
+            // URL showing where the source code can be accessed through the web browser
+            remoteUrl("https://github.com/sceneview/sceneview-android/blob/main/${project.name}/src/main/kotlin")
+            // Suffix which is used to append the line number to the URL. Use #L for GitHub
+            remoteLineSuffix.set("#L")
+        }
+    }
+}
+
+filamentTools {
+    // Material generation: .mat -> .filamat
+    materialInputDir.set(project.layout.projectDirectory.dir("src/main/materials"))
+    materialOutputDir.set(project.layout.projectDirectory.dir("src/main/assets/materials"))
+    iblInputDir.set(project.layout.projectDirectory.dir("src/main/environments"))
+    iblOutputDir.set(project.layout.projectDirectory.dir("src/main/assets/environments"))
```

Comment:
Could you add `// IBL and Skybox generation: .hdr -> _ibl.ktx and _skybox.ktx` before these lines?